scripts/dataset_processing/tts/preprocess_audio.py

```diff
@@ -97,6 +97,12 @@ def get_args():
     parser.add_argument(
         "--output_sample_rate", default=0, type=int, help="If provided, rate to resample the audio to."
     )
+    parser.add_argument(
+        "--output_format",
+        default=".flac",
```

Comment:
Double check if `.` brings in ambiguation in CLI. Is it better to default as `flac` instead?

Reply:
I changed it so it expects it without the ".", which is also consistent with soundfile.